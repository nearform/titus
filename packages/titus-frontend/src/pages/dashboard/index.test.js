import React from 'react'
import { render, cleanup } from '@testing-library/react'
import DashboardContainer from './index'

describe('<DashboardContainer />', () => {
  afterEach(cleanup)

  it('renders without crashing', () => {
    const { asFragment } = render(<DashboardContainer />)
    expect(asFragment()).toMatchSnapshot()
  })
})
