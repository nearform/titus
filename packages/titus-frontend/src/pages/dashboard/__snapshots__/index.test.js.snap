// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DashboardContainer /> renders without crashing 1`] = `
<DocumentFragment>
  <label
    class="hidden"
    for="language"
  >
    language:
  </label>
  <select
    class="language-selector"
    id="language"
  >
    <option
      value="en"
    >
      English 🇬🇧
    </option>
    <option
      value="ro"
    >
      Românā 🇷🇴
    </option>
    <option
      value="pt"
    >
      Português 🇵🇹
    </option>
  </select>
  <button
    class="logout button"
  >
    logout
  </button>
  <img
    alt="Titus logo"
    class="login-logo"
    src="img/Accel_Logo_Titus.svg"
  />
  <p>
    description
  </p>
  <a
    href="https://nf-titus.netlify.com/"
    rel="noopener noreferrer"
    target="_blank"
  >
    docs
  </a>
</DocumentFragment>
`;
